From 51abc6f5b1242dedf1288127d779d560883f58c5 Mon Sep 17 00:00:00 2001
From: VinceChang6637 <vince_chang@aspeedtech.com>
Date: Fri, 21 Jun 2024 16:56:52 +0800
Subject: [PATCH] revert limit to on chunk

Revert limit to on chunk commit.
https://github.com/openbmc/webui-vue/commit/740cbd525fa619e6a4190ec08c80f45ae72435eb

Using "LimitChunkCountPlugin " causes the following build error:
```
|  error  in webpack/runtime/runtimeId
|
| RuntimeIdRuntimeModule must be in a single runtime
```

This patch is a workaround to avoid build failures.

Another possible solution can be found at
https://github.com/webpack/webpack/issues/11431.

Upstream-Status: Inappropriate [ASPEED specific]
Signed-off-by: VinceChang6637 <vince_chang@aspeedtech.com>
---
 vue.config.js | 20 --------------------
 1 file changed, 20 deletions(-)

diff --git a/vue.config.js b/vue.config.js
index 0467ff5..4f6f4a1 100644
--- a/vue.config.js
+++ b/vue.config.js
@@ -1,6 +1,4 @@
 const CompressionPlugin = require('compression-webpack-plugin');
-const webpack = require('webpack');
-const LimitChunkCountPlugin = webpack.optimize.LimitChunkCountPlugin;
 
 module.exports = {
   devServer: {
@@ -51,16 +49,6 @@ module.exports = {
     }
   },
   configureWebpack: (config) => {
-    config.plugins.push(
-      new LimitChunkCountPlugin({
-        maxChunks: 1,
-      }),
-    );
-    config.optimization.splitChunks = {
-      cacheGroups: {
-        default: false,
-      },
-    };
     if (process.env.NODE_ENV === 'development') {
       config.devtool = 'source-map';
     }
@@ -108,14 +96,6 @@ module.exports = {
     config.experiments = {
       asyncWebAssembly: true,
     };
-
-    config.performance = {
-      hints: 'warning',
-      maxEntrypointSize: 512000,
-      maxAssetSize: 512000,
-    };
-
-    config.optimization.runtimeChunk = false;
   },
   pluginOptions: {
     i18n: {
-- 
2.25.1

